web/react-web/src/Content/Graph/Graph.js

```diff
@@ -142,9 +149,32 @@ class Graph extends Component {
                 name = `Active cases for ${one.long_name}`;
             }
 
+            let yRatio = 1.0;
+            switch (dataScaling) {
+                case SCALING_SAME_PEAK:
+                    let maxValue = one.max_value;
+                    if (one.type !== 'prediction') {
+                        maxValue = Math.max(...one.values);
+                    }
+                    yRatio = 1.0 / maxValue * 100.0;
+
+                    finalOptions.yAxis.title.text = 'Active cases %';
+                    finalOptions.yAxis.labels.format = '{value}%';
+                    break;
+                case SCALING_PER_CAPITA:
+                    yRatio = 1.0 / (one.population / 100000.0);
+                    finalOptions.yAxis.title.text = 'Active cases per 100 000 citizens';
+                    finalOptions.yAxis.labels.format = '{value}';
+                    break;
+                case SCALING_ABSOLUTE:
+                default:
+                    finalOptions.yAxis.title.text = 'Active cases';
+                    finalOptions.yAxis.labels.format = '{value}';
+                    break;
+            }
 
             const lineData = one.date_list.map((date, index) => {
-                const y = one.values[index];
+                const y = one.values[index] * yRatio;
```

Comment:
Is this the place where the value gets rounded to an integer? For countries with small numbers it would be better not showing rounded numbers but with perhaps one decimal place.

Alternatively bump the scale to cases per 1 million and we might not need any rounding.

Reply:
Rounding is actualy done when rendering tooltip. I'll change it to 1 000 000.